src/assets/styles/mapbox_styles.json

```diff
@@ -0,0 +1,39 @@
+[{
```

Comment:
It's pretty cool but also a bit magic (in a bad way) sometimes.

Depending on how often the data sources update, down the road we could take this a step further and upload the data to mapbox using the [js sdk](https://github.com/mapbox/mapbox-sdk-js) and style the whole thing using the free online editor. that would move all of this stuff out of the codebase. Plus the data would come in with the vector tiles which is pretty sweet.